src/main/java/AutoLotto/model/Lotto.java

```diff
@@ -4,10 +4,9 @@
 import java.util.List;
 import java.util.Set;
 
-import static autoLotto.model.LottoConstants.INVALID_LOTTO_NUMBERS;
-import static autoLotto.model.LottoConstants.VALID_LOTTO_LENGTH;
-
 public class Lotto {
+    private static final String INVALID_LOTTO_NUMBERS = "로또 번호의 개수는 6개이며, 각 번호는 1 이상 45 이하의 값만 가능합니다.";
+    private static final int VALID_LOTTO_LENGTH = 6;
     private Set<LottoNumber> lotto;
```

Comment:
재할당을 막기 위해 final 키워드를 추가하면 어떨까요?
클래스 변수와 멤버 변수 사이에 공백으로 가독성도 추가해 주시면 좋을 것 같아요
```suggestion

    private final Set<LottoNumber> lotto;
```